infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -128,23 +140,18 @@ public SequencerServer(ServerContext serverContext) {
         this.serverContext = serverContext;
         Map<String, Object> opts = serverContext.getServerConfig();
         this.executor = Executors.newFixedThreadPool(
```

Comment:
You can make this a single threaded executor and get rid of the variable (and merge the comments).

Reply:
Done.